### DISCUSSION TITLE
blog/exception-response

### DISCUSSION THREAD

**Discussion by giscus** [review-me-blog/review-me-blog.github.io#62] [blog comments]
# blog/exception-response

Intro

https://blog.review-me.page/blog/exception-response

**Comment by nayonsoso** [comment]
ref : https://mangkyu.tistory.com/204

**Comment by nayonsoso** [comment]
보강 : 
컨트롤러 메서드에서 예외가 발생하면, 스프링은 먼저 해당 컨트롤러 내의 @ExceptionHandler 메서드를 찾습니다.
없다면 @ControllerAdvice나 @RestControllerAdvice가 적용된 클래스의 @ExceptionHandler 메서드를 찾습니다.
그래도 없다면 최종적으로 BasicErrorController가 처리합니다.
